linux/keyman-config/km-config

```diff
@@ -12,14 +12,23 @@ if __name__ == '__main__':
         versionstring = "%s (package version %s)" % (__versionwithtag__, __pkgversion__)
     else:
         versionstring = "%s" % __versionwithtag__
-    parser = argparse.ArgumentParser(description='Keyman keyboards installation and information')
+    parser = argparse.ArgumentParser(description='km-config shows the currently installed ' +
+                                     'Keyman keyboard packages and allows you to view ' +
+                                     'information about them. It enables you to download new ' +
+                                     'keyboard packages from the website or install from ' +
+                                     'local files.')
     parser.add_argument('--version', action='version', version='%(prog)s version ' + versionstring)
     parser.add_argument('-v', '--verbose', action='store_true', help='verbose logging')
     parser.add_argument('-vv', '--veryverbose', action='store_true', help='very verbose logging')
     parser.add_argument('-i', '--install', action='store', help='download and/or install .kmp ' +
                         'package. INSTALL can either be a downloaded .kmp file, a file:// URL ' +
                         'pointing to a .kmp file, or a keyman:// URL, possibly with a ' +
                         'bcp47=<language> specified.')
+    parser.add_argument('url', nargs='?', default='', metavar='INSTALL',
```

Comment:
Would a command-line install look like this?
```
km-config url https://downloads.keyman.com/keyboards/sil_euro_latin/1.9.5/sil_euro_latin.kmp
```

Reply:
It's simply:

```
km-config https://downloads.keyman.com/keyboards/sil_euro_latin/1.9.5/sil_euro_latin.kmp
```